contents/plink/PLINK_Data_QC.ipynb

```diff
@@ -163,98 +163,12 @@
-      "Total genotyping rate is 0.996913.\n",
-      "556 variants and 247 people pass filters and QC.\n",
-      "Note: No phenotypes present.\n",
-      "--recode ped to MAF_less_5.ped + MAF_less_5.map ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n",
-      "PLINK v1.90b6.18 64-bit (16 Jun 2020)          www.cog-genomics.org/plink/1.9/\n",
-      "(C) 2005-2020 Shaun Purcell, Christopher Chang   GNU General Public License v3\n",
-      "Logging to MAF_greater_5_clean.log.\n",
-      "Options in effect:\n",
-      "  --file MAF_greater_5\n",
-      "  --geno 0.05\n",
-      "  --out MAF_greater_5_clean\n",
-      "  --recode\n",
-      "\n",
-      "7960 MB RAM detected; reserving 3980 MB for main workspace.\n",
-      ".ped scan complete (for binary autoconversion).41515161617171718181919192020212121222223232324242525252626272727282829292930303131313232333334343435353636363737383838393940404041414242424343444444454546464647474848484949505051515152525353535454555555565657575758585959596060616161626263636364646565656666676768686869697070707171727272737374747475757676767777787878797980808081818282828383848485858586868787878888898989909091919192929393939494959595969697979798989999100%\n",
-      "Performing single-pass .bed write (5868 variants, 247 people).\n",
-      "--file: MAF_greater_5_clean-temporary.bed + MAF_greater_5_clean-temporary.bim +9505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394%\n",
-      "MAF_greater_5_clean-temporary.fam written.\n",
-      "5868 variants loaded from .bim file.\n",
-      "247 people (125 males, 122 females) loaded from .fam.\n",
-      "Using 1 thread (no multithreaded calculations invoked).\n",
-      "Before main variant filters, 247 founders and 0 nonfounders present.\n",
-      "Calculating allele frequencies... 10111213141516171819202122232425262728293031323334353637383940414243444546474849505152535455565758596061626364656667686970717273747576777879808182838485868788899091929394959697989 done.\n",
-      "Warning: 6 het. haploid genotypes present (see MAF_greater_5_clean.hh ); many\n",
-      "commands treat these as missing.\n",
-      "Total genotyping rate is 0.996858.\n",
-      "2 variants removed due to missing genotype data (--geno).\n",
-      "5866 variants and 247 people pass filters and QC.\n",
-      "Note: No phenotypes present.\n",
-      "--recode ped to MAF_greater_5_clean.ped + MAF_greater_5_clean.map ... 101112131415161718192021222324252627282930313233343536373839404142434445464748495051525354555657585960616263646566676869707172737475767778798081828384858687888990919293949596979899done.\n"
-     ]
-    }
-   ],
    "source": [
-    "plink --file GWAS_clean_mind --maf 0.05 --recode --out MAF_greater_5\n",
+    "###### plink --file GWAS_clean_mind --maf 0.05 --recode --out MAF_greater_5\n",
```

Comment:
@hn929 no output for these? We need to retain all outputs if you could double check.